src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -213,7 +213,9 @@ def _get_next_sequencer(self, port, frequency, qubits: dict):
         # select the qubit with flux line, if present, connected to the specific port
         qubit = None
         for _qubit in qubits.values():
-            if _qubit.flux is not None and _qubit.flux.port == self.ports(port):
+            name = _qubit.flux.port.name
+            module = _qubit.flux.port.module
+            if _qubit.flux is not None and (name, module) == (port, self):
```

Comment:
I was also expecting that there is one port per `(module, name)` (so the two checks are equivalent), however this was not the case and `sequencer.qubit` was `None` for all sequencers before. I suspect there is another bug in how ports are created/handled, but here I wanted to do a minimal fix for the "urgent" issue.

Reply:
It truly makes the difference!
![image](https://github.com/qiboteam/qibolab/assets/10830876/4526531f-d39f-4d8b-b285-721534f3314f)
![image](https://github.com/qiboteam/qibolab/assets/10830876/2a5a6e57-e48e-404b-b0dd-308d57e24c58)

You can obtain the diff by running `python test819.py` with and without this change. The rest of the diff are white spaces (because comments are padded to stay to the right of the longest line).

So, the bug is "fixed" by this change (the requested instructions are now present).